docker-compose-non-dev.yml

```diff
@@ -92,10 +92,25 @@ services:
     healthcheck:
       disable: true
 
+  clickhouse:
```

Comment:
I suggest adding `superset-` as a prefix to follow the style to the file.

Reply:
I do not agree here, because additional services do not have a prefix.